files/default/solr-logrotate

```diff
@@ -0,0 +1,10 @@
+/var/log/solr/*log {
+    daily
+    rotate 10
+    missingok
+    notifempty
+    compress
+    postrotate
```

Comment:
convert this from GI
```
    scripts:
      postrotate: |
        BUCKET={{ LogBucketName }}
        INSTANCE_ID=`curl --silent http://169.254.169.254/latest/meta-data/instance-id`
        BOOT_TIME=`who -b | cut -d' ' -f13,14 --output-delimiter=::`
        aws s3 cp /apps/jboss/server/default/log/ s3://${BUCKET}/jboss/${BOOT_TIME}::${INSTANCE_ID}/ \
          --recursive --sse AES256 --exclude "*"  --include "server.log*"
        aws s3 cp /apps/jboss/server/default/log/ s3://${BUCKET}/jboss/${BOOT_TIME}::${INSTANCE_ID}/ \
          --recursive --sse AES256 --exclude "*"  --include "boot.log*"
```

so that when it rotates it copies the files to s3 every day or hour.

Reply:
OK, we should be able to implement this just by reusing our `archive-logs.sh` script.